### PR TITLE
Kaina 7

### PR DESCRIPTION
**Pull request by kaina1129**

### CODE REVIEW

**Comment by mxdi9i7** [comment]
image

**Comment by mxdi9i7** [comment]
image

**Comment by mxdi9i7** [comment]
image

**Comment by mxdi9i7** [comment]
image

**Comment by mxdi9i7** [comment]
very good job! beside some minor style issues, very well done

File: week7/kaina/netflix/src/components/Banner/index.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		h3 {
+			font-size: 27px;
+			text-transform: uppercase;
+			margin: 0px;
+			color: $white;
+			display: inline;
+		}
+		span {
+			text-transform: uppercase;
+			font-size: 27px;
```
Comment (chuxu200328) [inline week7/kaina/netflix/src/components/Banner/index.scss]: Do not use 27px

File: week7/kaina/netflix/src/components/Category/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+						<div className='image-box'>
+							<img src={Pink} alt='image' />
+							<h3>Romance</h3>
+							<i class='fas fa-chevron-down'></i>
+						</div>
+						<div className='image-box'>
+							<img src={Green} alt='image' />
+							<h3>Thriller</h3>
+
+							<i class='fas fa-chevron-down'></i>
```
Comment (chuxu200328) [inline week7/kaina/netflix/src/components/Category/index.js:46]: Rewrite this with the component. by passing in data props. Remember to stay DRY

File: week7/kaina/netflix/src/components/Category/index.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	padding: 16px 0px 0px 60px;
+	position: relative;
+	top: -90px;
+
+	.category-box {
+		overflow: hidden;
+		width: 100%;
+
+		h3 {
+			font-size: 23px;
```
Comment (chuxu200328) [inline week7/kaina/netflix/src/components/Category/index.scss]: 23px font-size does not make sense.

**Comment by chuxu200328** [comment]
image
background repeat

**Comment by chuxu200328** [comment]
image

File: week7/kaina/netflix/src/components/Banner/index.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+		display: flex;
+		font-size: 16px;
+		.lecture {
+			display: flex;
+			align-items: center;
+			font-weight: 600;
+			svg {
+				margin: 0px 10px 0px 36px;
+			}
+			margin-right: 87px;
```
Comment (chuxu200328) [inline week7/kaina/netflix/src/components/Banner/index.scss]: 87px is not a common number

File: week7/kaina/netflix/src/components/Navbar/index.scss [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+			i {
+				font-size: 22px;
+			}
+			span {
+				background-color: $red;
+				border-radius: 50%;
+				position: relative;
+				right: 11px;
+				bottom: 10px;
+				padding: 1px 6px;
```
Comment (chuxu200328) [inline week7/kaina/netflix/src/components/Navbar/index.scss:70]: 1px? what's the reason for this?

**Review by chuxu200328 — COMMENTED** [review]
Nice work! Just remember do not use uncommon numbers(for example: 19, 23, 91).

**Comment by kaina1129** [comment]
> image
> background repeat

updated nonrepeat

File: week7/kaina/netflix/src/components/Banner/index.scss (hunk 1 above)
Comment (knazhang) [reply week7/kaina/netflix/src/components/Banner/index.scss]: updated

File: week7/kaina/netflix/src/components/Category/index.scss (hunk 3 above)
Comment (knazhang) [reply week7/kaina/netflix/src/components/Category/index.scss]: updated

File: week7/kaina/netflix/src/components/Banner/index.scss (hunk 4 above)
Comment (knazhang) [reply week7/kaina/netflix/src/components/Banner/index.scss]: updated

File: week7/kaina/netflix/src/components/Navbar/index.scss (hunk 5 above)
Comment (knazhang) [reply week7/kaina/netflix/src/components/Navbar/index.scss:70]: top and bottom have 1 px of padding

File: week7/kaina/netflix/src/components/Category/index.js (hunk 2 above)
Comment (knazhang) [reply week7/kaina/netflix/src/components/Category/index.js:46]: ok

**Comment by chuxu200328** [comment]
image
image
fix these gaps

**Review by chuxu200328 — APPROVED** [review]

**Pull request merged**